DendraScience/dendra-web-ui
# feat:User CRUD

## Related Issues:
[USERS CRUD](https://github.com/orgs/DendraScience/projects/1/views/1?pane=issue&itemId=21895579)
Resolves #30 #36 #37

## Main Changes:

- Users list
- User create || if person without same email create new person with given data
- User update
- User delete

## Steps To Test:

1. Login 
2. Navigate to http://localhost:3000/users
3. Check users list, create, update and delete
